holoviews/plotting/util.py

```diff
@@ -174,6 +174,47 @@ def compute_overlayable_zorders(obj, path=[]):
     return zorder_map
 
 
+def is_dynamic_overlay(dmap):
+    """
+    Traverses a DynamicMap graph and determines if any components
+    were overlaid dynamically (i.e. by * on a DynamicMap).
+    """
+    if not isinstance(dmap, DynamicMap):
+        return False
+    elif dmap.callback._is_overlay:
+        return True
+    else:
+        return any(is_dynamic_overlay(dm) for dm in dmap.callback.inputs)
+
+
+def split_dmap_overlay(obj, depth=0):
+    """
+    Splits a DynamicMap into the original component layers it was
+    constructed from.
+    """
```

Comment:
I would consider creating an issue discussing the limitations/problems with this utility and referencing it in the docstring. It might be a useful reminder for the next person looking at this code.

Reply:
I'm not aware of any limitations, but I'm happy to expand the docstring.